src/Microsoft.OpenApi.OData.Reader/Operation/OperationHandler.cs

```diff
@@ -44,15 +44,15 @@ public virtual OpenApiOperation CreateOperation(ODataContext context, ODataPath
             // Security
             SetSecurity(operation);
 
+            // Parameters
```

Comment:
The place of function matters here?

Reply:
Yes. It needs to be processed before `SetResponses()` so that we can use the parameters in the argument [here](https://github.com/irvinesunday/OpenAPI.NET.OData/blob/a72873cf6170c49785c82f4a6ad29c1757376aef/src/Microsoft.OpenApi.OData.Reader/Operation/NavigationPropertyGetOperationHandler.cs#L97) when calling the `CreateLinks()` method; which will get used [here](https://github.com/irvinesunday/OpenAPI.NET.OData/blob/a72873cf6170c49785c82f4a6ad29c1757376aef/src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiLinkGenerator.cs#L50-L57).